configserver/src/main/java/com/yahoo/vespa/config/server/session/SessionRepository.java

```diff
@@ -320,28 +276,21 @@ void activate(RemoteSession session) {
         log.log(Level.INFO, session.logPre() + "Session activated: " + sessionId);
     }
 
-    public void delete(Session remoteSession) {
-        long sessionId = remoteSession.getSessionId();
-        // TODO: Change log level to FINE when debugging is finished
-        log.log(Level.INFO, () -> remoteSession.logPre() + "Deactivating and deleting remote session " + sessionId);
-        deleteRemoteSessionFromZooKeeper(remoteSession);
-        remoteSessionCache.remove(sessionId);
-        LocalSession localSession = getLocalSession(sessionId);
-        if (localSession != null) {
-            // TODO: Change log level to FINE when debugging is finished
-            log.log(Level.INFO, () -> localSession.logPre() + "Deleting local session " + sessionId);
-            deleteLocalSession(localSession);
-        }
+    public void delete(Session session) {
+        long sessionId = session.getSessionId();
+        deleteSessionFromZooKeeper(session);
+        sessionCache.remove(sessionId);
+        deleteLocalSession(session);
```

Comment:
Perhaps rename this method as well, to be in line with others? `deleteSessionFromFileSystem`?

Reply:
Yeah, there is more renaming, collapsing and simplification that can be done. Will follow up in a later PR